### DISCUSSION TITLE
Save frame capture to file on alert?

### DISCUSSION THREAD

**Discussion by rotx** [nzymedefense/nzyme#475] [Ideas]
I installed this on a RasPi and wanted to thank you for publishing this software.
My main access point is a Ubiquiti UAP nanoHD (802.11ac Wave2) running on the latest official firmware (5.43.36.12724 as of this writing). There are two issues with this access point that I have a hard time explaining:
1. As discussed in another post, the access point "spuriously" advertises on all 2.4 GHz channels. This does not seem to happen on the 5 GHz channels!?
2. The beacon rate, while typically low, will (again, "spuriously") go up to as high as 122.
I have not found anything the the Unifi controller that would allow me to configure this behavior. If anyone knows how, please let me know.

As an idea for a new feature, would it be possible to save the frame that caused the alert into pcap (or other common capture format) for later analysis? A feature like this might allow us to figure out why a piece of hardware is behaving the way it is and/or submit a proper bug report and/or save for forensic analysis.
Thanks :)

**Comment by lennartkoopmann** [comment]
I created feature request #560 from this. Very good idea.
